cmd/node/node_test.go

```diff
@@ -566,8 +546,10 @@ func TestSpacemeshApp_NodeService(t *testing.T) {
 		poetHarness.HTTPPoetClient, clock, h, edSgn)
 	require.NoError(t, err)
 
+	appCtx, appCancel := context.WithCancel(context.Background())
```

Comment:
In a test I usually just immediately call `defer appCancel()` after creating a `context`. To avoid it not being cleaned up if the test fails.

Reply:
done